src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Consumer.SqlServer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+using System.Reflection;
+using System.Resources;
+using System.Runtime.InteropServices;
+
+[assembly: ComVisible(false)]
+[assembly: AssemblyProduct("Vlaamse basisregisters")]
+[assembly: AssemblyDescription("Message handling with Kafka.")]
```

Comment:
@ArneD I copied the description from paket.template, but it was the same for all Kafka projects.